### PR TITLE
2022 3 amxfmx

### PR DESCRIPTION
**Pull request by JunAishima**
small changes done during the cycle

- puck ordering
- c3d location
- Edwin's MT_LoadCheck

### CODE REVIEW

File: bin/mt-load-check [hunk 1]
```diff
@@ -12,12 +12,12 @@ esac
 case $servorws in
 	ws*)
 		echo Workstation - activating GUI env
-		conda activate lsdc-gui-2022-1.1
+		PYTHONTOUSE=/opt/conda_envs/lsdc-gui-2022-1.1/bin/python
```
Comment (vshekar) [inline bin/mt-load-check:15]: An environment variable name like PYTHONENV or PYTHON_ENV may be more readable

Comment (JunAishima) [reply bin/mt-load-check:15]: I agree - looking back at this, at first glance, I saw it as PYTHONHOUSE!

Comment (JunAishima) [reply bin/mt-load-check:15]: would you be happy with PYTHON_BINARY or CONDA_ENV_PYTHON_BINARY? or is there something else that you think would be better?

Comment (JunAishima) [reply bin/mt-load-check:15]: ok, going with `CONDA_ENV_BASE`, and bin/python will be appended to the directory name to define the python binary location

**Review by vshekar — APPROVED** [review]

**Comment by JunAishima** [comment]
ok, now I understand what happened... I had made the changes onto NSLS-II/2022-3-amxfmx but then never merged that branch into master. that's why I remembered these changes having been approved in a pull request.

**Pull request merged**